app/controllers/partnering_organizations_controller.rb

```diff
@@ -1,4 +1,4 @@
-class PartneringOrganizationsController < ApplicationController
+class PartneringOrganizationsController < Devise::RegistrationController 
```

Comment:
the entire PartneringOrganizationsController should not inherit from the Devise Controller. the devise generated controllers should be used for authentication. so make your changes in partnering_organization_devise/registrations_controller.rb